group_vars/jenkins_server/vars.yml

```diff
@@ -87,10 +92,7 @@ jenkins_url: 'http://localhost:8080/'
 jenkins_cli_dest: '/opt/jenkins-cli.jar'
 jenkins_xml_dest: '/var/lib/jenkins/xml_files/'
 jenkins_projects: 
-  - 'AGCT'
```

Comment:
Waarom zijn AGCT, NGS_Demultiplexing, NGS_Automated eruit gehaald?

Reply:
Hier hebben we het een keer over gehad en toen bleek dat het niet wenselijk was om deze pipelines in Jenkins te hebben. Ze zitten ook niet in de huidige Jenkins. Mocht blijken dat dit toch wel gewenst is, dan kunnen we ze altijd toevoegen.